hypervisor/src/kvm/mod.rs

```diff
@@ -2263,6 +2272,20 @@ impl cpu::Vcpu for KvmVcpu {
             Ok(_) => Ok(()),
         }
     }
+
+    #[cfg(target_arch = "x86_64")]
+    ///
+    /// Trigger NMI interrupt
+    ///
+    fn nmi(&self) -> cpu::Result<()> {
+        // SAFETY:  Safe because we know that our file is a KVM fd and that the request is one of
+        // the ones defined by kernel.
+        let ret = unsafe { ioctl(&self.fd, KVM_NMI()) };
```

Comment:
Why not use the API from kvm-ioctl to raise NMI like we do at other places? https://github.com/rust-vmm/kvm-ioctls/commit/111f11a43984c45f3b4dbd1e9be3ed9aa1407963

Reply:
Yeah, I searched the kvm-ioctl source and didn't find the API of nmi() when I did this work.
After looking into the git log of kvm-ioctl I find that https://github.com/rust-vmm/kvm-ioctls/commit/111f11a43984c45f3b4dbd1e9be3ed9aa1407963 is merged into main branch last week, and Cloud Hyperviosr now uses the version 0.13.0 of kvm-ioctl which starts at April.
Anyway, I'm glad to turn into the new API after we bump the kvm-ioctl repo to newer version. WDYT?

Reply:
ack

Reply:
Since the new version is now part of CLH. We can use the API from kvm-ioctl instead.

Reply:
@up2wing I think this one still needs fixing.

Reply:
done